src/app/components/splitbutton/splitbutton.spec.ts

```diff
@@ -21,8 +22,16 @@ describe('SplitButton', () => {
           SplitButton,
         ],
       });
-      
+
       fixture = TestBed.createComponent(SplitButton);
       splitbutton = fixture.componentInstance;
     });
+
+    it('should accept readonly options', () => {
+        const models : MenuItem[] = [
+            {label:'home', url:'https://www.primefaces.org/primeng/'},
+            {label:'split-button', url:'https://www.primefaces.org/primeng/#/splitbutton'}
+        ];
+        splitbutton.model = models;
+    });
```

Comment:
Did you forget `as const` here?

Reply:
Yes, it's an error from me. Good catch :)

I'll be able to correct it in few hours

Reply:
Resolved with 3ebce6a